test/regress/regress3/bv_to_int_check_ne_bvlshr0_4bit.smt2

```diff
@@ -0,0 +1,19 @@
+; COMMAND-LINE:  --solve-bv-as-int=bv 
+(set-logic BV)
```

Comment:
Seems to be missing a status.

Reply:
Note that regress3 is not checked by CI, so you should maybe ensure `make regress3` works locally.

Reply:
this file was actually redundant and did not appear in CMakeLists. fixed now. `ctest -R bv_to_int` passes locally.